src/app/modules/customer-management/components/projects-type/components/create-project-type/create-project-type.component.html

```diff
@@ -1,11 +1,31 @@
 <div class="container">
-  <form style="width: 600px;">
+  <form style="width: 600px;" [formGroup]="projectTypeForm" (ngSubmit)="onSubmit(projectTypeForm.value)">
     <div class="form-group">
-      <input type="text" class="form-control form-control-sm" id="" aria-describedby="" placeholder="Name" />
-      <textarea class="form-control form-control-sm mt-2" id="descriptionTextArea" rows="3"
-        placeholder="Description"></textarea>
+      <input
+        type="text"
+        class="form-control form-control-sm"
+        id="name"
+        formControlName="name"
+        aria-describedby=""
+        [class.is-invalid]="name.invalid && name.touched"
+        required
+        placeholder="Name"
+      />
+      <div class="text-danger" *ngIf="(name.dirty || name.touched) && name.invalid && name.errors.required">
+        Name is required.
+      </div>
+      <textarea
+        class="form-control form-control-sm mt-2"
+        id="descriptionTextArea"
+        rows="3"
+        placeholder="Description"
+        formControlName="description"
+      ></textarea>
```

Comment:
we can set a default value for optional fields, that way we can avoid the error when we do two continue creations only with required fields